packages/api/src/queue-processor.ts

```diff
@@ -49,6 +49,14 @@ export const getBackendQueue = async (): Promise<Queue | undefined> => {
   }
   backendQueue = new Queue(QUEUE_NAME, {
     connection: redisDataSource.workerRedisClient,
+    defaultJobOptions: {
+      backoff: {
+        type: 'exponential',
+        delay: 2000, // 2 seconds
+      },
+      removeOnComplete: true,
```

Comment:
Have to test but this could mess up the bullmq UI

Reply:
yeah, how about we keep jobs based on their age?

for example, we could only keep failed jobs to 24 hours so we can avoid out of memory error and save costs

Reply:
Yeah agreed. I think it does do that now, but I'm not exactly sure what the default parameters are.

Reply:
let me set the parameters explicitly and test

Reply:
@jacksonh i think by default bullmq keeps all the completed and failed jobs so here i change it to only keep completed jobs for 24 hours and failed job for 7 days

i think the bullmq ui will be messed up once this is live